pkg/interactor/cloud.go

```diff
@@ -0,0 +1,41 @@
+package interactor
+
+import (
+	"errors"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/network/cloud"
+)
+
+var ErrTokenMissing = errors.New("no access token provided")
+
+// CloudInteractor ...
+type CloudInteractor struct {
+	cloud cloud.Cloud
+}
+
+// NewCloudInteractor ...
+func NewCloudInteractor(cloud cloud.Cloud) CloudInteractor {
+	return CloudInteractor{cloud}
+}
```

Comment:
Constructors must return a ptrrrrr, jesus :pray:

Reply:
Take it easy, man

Reply:
This interactor was removed :grimacing: